<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-N2GHRU3U" failures="2" tests="4" name="Test" time="19.403" errors="0" timestamp="2021-04-28T15:58:03 IST">
  <testcase classname="ListenerDemo.SecondTestCaseListener" name="googleVerifySecondTC" time="5.521"/>
  <testcase classname="ListenerDemo.SecondTestCaseListener" name="testFailMessageSecondTC" time="5.032">
    <failure type="java.lang.AssertionError" message="expected [Yahoo] but found [Google]">
      <![CDATA[java.lang.AssertionError: expected [Yahoo] but found [Google]
at ListenerDemo.SecondTestCaseListener.testFailMessageSecondTC(SecondTestCaseListener.java:36)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- testFailMessageSecondTC -->
  <testcase classname="ListenerDemo.FirstTestCaseListener" name="googleVerify" time="4.892"/>
  <testcase classname="ListenerDemo.FirstTestCaseListener" name="testFailMessage" time="3.898">
    <failure type="java.lang.AssertionError" message="expected [Yahoo] but found [Google]">
      <![CDATA[java.lang.AssertionError: expected [Yahoo] but found [Google]
at ListenerDemo.FirstTestCaseListener.testFailMessage(FirstTestCaseListener.java:36)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- testFailMessage -->
</testsuite> <!-- Test -->
